app/assets/locales/messages_ccodk_default.txt

```diff
@@ -304,6 +304,8 @@ demo.mode.feature.unavailable=Sorry, this app uses a feature that is not availab
 
 select.search.label=Search:
 select.search.status=${0} of ${1} results for your search: "${2}"
+select.callout.search.status=${0} of ${1} results for callout result

```

Comment:
This string seems awkwardly worded to me, I'm not sure what it's trying to convey.